static/app/utils/replays/getCurrentUrl.tsx

```diff
@@ -17,14 +17,14 @@ function getCurrentUrl(
   ) as BreadcrumbTypeNavigation[];
 
   const initialUrl = replayRecord.tags.url;
-  const origin = initialUrl ? new URL(initialUrl).origin : '';
+  const origin = initialUrl ? new URL(initialUrl).origin : replayRecord.urls[0] ?? '';
```

Comment:
I think initialUrl should be what you had previously (`urls[0]`), and `origin` remains the same.

Also if `initialUrl` is invalid URL here, it will throw

Reply:
Are we not capturing the url tag anymore? From recent replays it looks like we're not. I've been looking through SDK source this morning trying to figure out whats what because the getCurrentUrl tests need to be updated as well. Might need to update the mocks

Reply:
Do you have access to this? https://github.com/getsentry/replay-backend/blob/main/blueprints/api.md#replays-organizationsorganization_slugreplays -- we don't capture it as a tag because we collect a list of URLs for the replay so it's a bit redundant.